website/docs/tutorials/07-testing/02-cheatcodes/example.md

```diff
@@ -0,0 +1,44 @@
+# `example`
+
+```python
+def example(self, *args: Any, **kwargs: Any) -> None:
+```
+
+This is an addition to the [`given`](./given.md) cheatcode, but, due to how the protostar works,
+`given` cheatcode does not have to be used in order to invoke `example`.
+While `given` runs tests with multiple, randomly generated data sets,
+`example` takes certain values and runs tests using them.
+Tests with the data specified in `example` will always be run before applying the data from `given`.
+
+Multiple `example`s can be used for one test. In such a case, all of them will be run sequentially,
+only after that allowing `given` to run.
+
+`example` is not limited by [`max_examples`](./max-examples.md) and is not connected to it in any way.
+
+:::warning
+This cheatcode is only available in [setup cases](../README.md#setup-case).
+:::
```

Comment:
I wouldn't present it as an addition to `given` cheatcode. It is a standalone feature. Please make the first sentence explanation what the cheatcode does as it is the most important information. 
I provided an simplified description, lmk what do you think

```suggestion
Parametrizes test with explicitly provided data. You can provide multiple examples for one test. In such a case, they run sequentially.

Even when examples are provided protostar still runs fuzz tests against the test case. Protostar does it after running all of the examples.

:::warning
This cheatcode is only available in [setup cases](../README.md#setup-case).
:::
:::warning
`example` is not limited by [`max_examples`](./max-examples.md) and is not connected to it in any way.
:::

```

Reply:
> Even when examples are provided protostar still runs fuzz tests against the test case. Protostar does it after running all of the examples.

Actually, I handled this in the latest commit and it doesn't work like that anymore. Since we want to make `example` a standalone cheatcode, it should be possible to run it without `given` and just get the examples we want. Thanks for the rest, I will come up with a new version.

Reply:
[update docs](https://github.com/software-mansion/protostar/pull/947/commits/75b02de8f52d7eced8f64ae8687db4947c59bb45)